Doc: Data model schema (#536)
aristanetworks/ansible-cvp: ansible_collections/arista/cvp/docs/schema/cv_container_v3.md

# Schema for cv_container_v3

| Variable | Type | Required | Default | Choices | Description |
| -------- | ---- | -------- | ------- | ------------------ | ----------- |

> | apply_mode | str | No | loose | loose<br>strict | Set how configlets are attached/detached on container. If set to strict all configlets not listed in your vars are detached. |

Reviewer: nit: `Set how configlets are attached/detached to containers. If set to strict, all configlets not listed in your vars will be detached`

> | apply_mode | str | No | loose | loose<br>strict | Set how configlets are attached/detached on container. If set to strict all configlets not listed in your vars are detached. |

Author: ```suggestion
| apply_mode | str | No | loose | loose<br>strict | Set how configlets are attached/detached to containers. If set to strict, all configlets not listed in your vars will be detached. |
```

> | state | str | No | present | present<br>absent | Set if ansible should build or remove devices on Cloudvision |

Reviewer 2: ```suggestion
| state | str | No | present | present<br>absent | Set if Ansible should build or remove devices on CloudVision |
```

> | topology | dict | Yes |  |  | Yaml dictionary to describe intended containers |

Reviewer 2: ```suggestion
| topology | dict | Yes |  |  | YAML dictionary to describe intended containers |
```
